server/pbench/bin/pbench-base.sh

```diff
@@ -65,7 +65,7 @@ fi
 mail_recipients=$(getconf.py mailto pbench-server)
 
 # make all the state directories for the pipeline and any others needed
-LINKDIRS="TODO TO-COPY-SOS TO-SYNC SYNCED TO-LINK TO-INDEX INDEXED WONT-INDEX DONE BAD-MD5"
+LINKDIRS="TODO TO-COPY-SOS TO-SYNC SYNCED TO-LINK TO-INDEX INDEXED WONT-INDEX DONE BAD-MD5 SATELLITE-MD5-PASSED SATELLITE-MD5-FAILED TO-DELETE SATELLITE-DONE"
```

Comment:
We should vet this list: I think it's missing some. Let's check it against the state diagram.